mrp_lot_expiry_date/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module extends the functionality of mrp:
+
+  * Compute expiration date of mrp lot based on scheduled date
```

Comment:
planned production date

Reply:
Let me revisit this because I am probably looking at the wrong date field

Reply:
Probably not, as it the screenshot it clearly shows scheduled_date, which is date_planned_start. Of course we can base this on date_planned_end, which is computed like https://github.com/OCA/OCB/blob/16.0/addons/mrp/models/mrp_production.py#L688

Reply:
No, it's actually the right field. Btw i didnt think you had the wrong field, was just suggesting a clearer text, to my eyes "scheduled production date" or "planned production date" is clearer than "scheduled date" but maybe it's just me.

We could also extend this text a bit to explain about the boolean etc, now that we have it more clear. Or we could wait for client approval first.